plugins/scaffolder-backend-module-cookiecutter/src/actions/fetch/cookiecutter.ts

```diff
@@ -233,6 +248,8 @@ export function createFetchCookiecutterAction(options: {
         _copy_without_render: ctx.input.copyWithoutRender,
         _extensions: ctx.input.extensions,
         imageName: ctx.input.imageName,
+        templateDir: templateDir,
+        templateContentsDir: templateContentsDir,
```

Comment:
I think think there's any reason to pass these through `values`, we might as well pass them as separate options to `run({...})` right? The values are what we end up passing to cookiecutter for templating, and while we do deconstruct and remove these from that set of values, you can still end up in confusing naming clashes.

If that's the case we should move over the `imageName` too. Will save us from all the `as string` casting as well :grin:

Reply:
Agreed. Updated the code to not pass everything in through values.